setup.py

```diff
@@ -4,7 +4,7 @@
     Instructions:
     # Build:
     rm -rf dist/ omegaconf.egg-info/
-    python setup.py sdist bdist_wheel
+    python -m build
```

Comment:
I copied this from https://hydra.cc/docs/development/release

Reply:
This is fine as long as it works correctly. (It took some effort to get it to work for Hydra, but it's a more complicated project).

Reply:
On my local machine it is working without issue.